exporter/datadogexporter/examples/collector.yaml

```diff
@@ -48,6 +48,28 @@ receivers:
   #       static_configs:
   #       - targets: ['0.0.0.0:8888']
 
+  # Read more about filelog receiver https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver .
+  filelog:
+    include_file_path: true
+    poll_interval: 500ms
+    include:
+      # Logs will be collected from the below file path. Please ensure it is correct or edit as needed.
+      # filelog receiver by default ends from end of the file.
+      - /var/log/**/*example*/*.log
+     # Depending on the format of the logs it may be recommended to set up a parser.
```

Comment:
This is better. Please make sure to not miss the [other comments](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14795#discussion_r993056578) which have now disappeared. This is still ambiguous. You say "depending on the format of the logs" or that "it may be recommended" but the example is clearly the path for JSON logs, so instead maybe say:
```yaml
# If you're using JSON logs, use the below parser ...
```

Reply:
Updated , can u verify